build.xml

```diff
@@ -2025,6 +2125,9 @@
         </classpath>
     </taskdef>
     <mkdir dir=".settings" />
+    <echo file=".settings/org.eclipse.jdt.core.prefs"><![CDATA[eclipse.preferences.version=1
```

Comment:
Needed because high-scale-lib provides a java.util package which causes eclipses to prevent compilation because the java.util module is detected in the JDK and on the classpath.

![Screenshot From 2025-03-18 00-16-46](https://github.com/user-attachments/assets/d0452186-764c-4407-869c-861e59a8c499)